client/src/components/player-list/UserScoreList.js

```diff
@@ -43,7 +43,11 @@ const buildUserList = (userScores) => {
 
 const UserScoreList = ({ userScores }) => {
   console.log(userScores);
-  return <List component="nav">{buildUserList(userScores)}</List>;
+  return (
+    <div className="userScoreList">
```

Comment:
@mahadevans87 : Why do we have this extra div in UserScoreList and LogWindow ?

Reply:
Ah. This is an issue with the Material UI Grid. If we apply a css with margins (or any properties that are set by the Grid Component), MUI's Grid takes priority over our classes. Not sure if there is a better way

Reply:
Okay.